src/main/java/com/artipie/VertxMain.java

```diff
@@ -194,7 +194,7 @@ private void startRepos(final Settings settings, final Metrics metrics) {
         final String yaml = ".yaml";
         final Collection<RepoConfig> configs = storage.list(Key.ROOT).thenApply(
             keys -> keys.stream().map(Key::string)
-                .filter(name -> name.endsWith(yaml))
+                .filter(name -> name.endsWith(yaml) && name.charAt(0) != '_')
```

Comment:
@olenagerasimova it seems that this change is not really related to other changes in this PR

Reply:
@olegmoz hm, please, check #798 description - I've described this bug there, so I think it can be make here

Reply:
@olenagerasimova I see it in the issue now, yet do not understand how these bug fixes are related to replacing existing classes with api-management library. Especially the one with underscore names. If we don't allow names starting with underscore, then with should document it

Reply:
@olegmoz dashboard with `org` layout cannot be used while any of these bugs are valid - I faced them trying to start artipie and access any dashboard page locally. 
I'd suggest to properly deal with underscore names (or catching exceptions with invalid repo configs, whatever we decide) in the separate issue/PR.

Reply:
@olenagerasimova got it, thanks